profile_library/eq-3/HM-ES-PMSw1-DR/model.json

```diff
@@ -0,0 +1,20 @@
+{
+  "measure_description": "Manually measured",
+  "measure_method": "manual",
+  "measure_device": "From manufacturer specifications",
+  "name": "HM-ES-PMSw1-DR",
+  "standby_power": 0.3,
```

Comment:
You are sure the power consumption is same with the relay turned on and off?
All other profiles of smart switches which are in the DB consume more when the relay is turned on, it needs to power the magnetic field so it must be a little higher.

Reply:
It is not easy to measure since the current is so low that the error margin of all consumer measureing products is larger than the difference will be. 
That is also the reason why I relyed on the manufacturer documents and did not measure. This is not a chaep product but from a German manufacturer. I am pretty sure that the specs are correct.

I will still do the efford and try to measure it with my Amp meter clamp-tool.

Reply:
Ah I see it's a DIN rail switch module. Not so easy to measure. To measure `standby_power_on` also no consumers must be connected to the switch. So you'll need to make changes to the wiring which is unfeasable.
Also I don't think clamp tool will be accurate enough.
So let's merge.

Reply:
Yes it is a DIN rail.
Similar will be all the other devices I will add soon than. They are either wall mounted switches or junction box actors. All of them from the same manufacturer.

If I get around measuring `standy_power_on` current draw, I will update my submits if I can find any solid readings at that point.